### PR TITLE
Added Snakes and Ladders Game

### PR DESCRIPTION
**Pull request by vaishnavicode**
Added Snakes and Ladders Game which prints output in console

### CODE REVIEW

File: src/App.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class App {
```
Comment (VivekKumarSingh07) [inline src/App.java:1]: Delete this file If not in use

File: src/Main.java [hunk 2]
```diff
@@ -19,6 +19,8 @@ public static void main(String[] args) {
 
                 switch (choice) {
                     case 1:
+                        SnakesAndLadders obj = new SnakesAndLadders();
+                        obj.snakesAndLadders();
                         break;
 
                     case 2:
```
Comment (VivekKumarSingh07) [inline src/Main.java:26]: Remove these cases, if not in use or write the comments

File: src/SnakesAndLadders.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    //Method to start the snakes and ladders game
+    public void snakesAndLadders() {
+
+        //Initializing the game over variable to false
+
+        gameOver = false;
+
+        //Initializing the Ladders and Snakes array
+        Ladders = new int[]{1,4,8,21,28,50,71,80};
```
Comment (VivekKumarSingh07) [inline src/SnakesAndLadders.java:24]: Can this be converted into class as they have properties related to it i.e. head or tail and initialise that?

File: src/SnakesAndLadders.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                if(sc.nextInt() == i+1){
+                    playerPositions[i] = snakesAndLaddersMove(playerPositions[i]);
+                    System.out.println("Player " + (i+1) + " current position: " + playerPositions[i]);
+                    
+                }
+                else{
+                    System.out.println("Invalid input. Please try again.");
+                    i--;
+                }
+
```
Comment (VivekKumarSingh07) [inline src/SnakesAndLadders.java:64]: Uneven spacing, do add extensions in VS code to remove that.

**Review by VivekKumarSingh07 — CHANGES_REQUESTED** [review]
1. Remove temp files (vscode or class files) using gitignore
2. Use code formatter
3. Move the variable to appropriate classes.

File: src/Main.java (hunk 2 above)
Comment (vaishnavicode) [reply src/Main.java:26]: Done

File: src/App.java (hunk 1 above)
Comment (vaishnavicode) [reply src/App.java:1]: File deleted.

**Pull request closed without merging**